examples/reviews/implementations/aggregators/vocab.py

```diff
@@ -2,7 +2,6 @@ def aggregate_spark(data, columns, args):
     import pyspark.sql.functions as F
     from pyspark.ml.feature import StopWordsRemover, RegexTokenizer
 
-    input_data = data.withColumn(columns["col"], F.lower(F.col(columns["col"])))
     regexTokenizer = RegexTokenizer(inputCol=columns["col"], outputCol="token_list", pattern="\\W")
     regexTokenized = regexTokenizer.transform(data)
 
```

Comment:
Can you use the nltk stop word list here too? [StopWordsRemover()](https://spark.apache.org/docs/latest/api/python/pyspark.ml.html#pyspark.ml.feature.StopWordsRemover) takes an arg to override the list.